includes/documents/abstract-wcpdf-order-document-methods.php

```diff
@@ -1282,36 +1282,25 @@ public function document_notes(): void {
 		}
 	}
 
-	public function document_display_date() {
+	public function document_display_date(): string {
 		$document_display_date = $this->get_display_date( $this->get_type() );
 
-		//If display date data is not available in order meta (for older orders), get the display date information from document settings order meta.
+		// If display date data is not available in order meta (for older orders), get the display date information from document settings order meta.
 		if ( empty( $document_display_date ) ) {
-			$document_settings = $this->settings;
-			if( isset( $document_settings['display_date'] ) ) {
-				$document_display_date = $document_settings['display_date'];
-			}
-			else {
-				$document_display_date = 'invoice_date';
-			}
+			$document_settings     = $this->settings;
+			$document_display_date = $document_settings['display_date'] ?? 'document_date';
 		}
 
-		$formatted_value = $this->get_display_date_label( $document_display_date );
-		return $formatted_value;
+		return $this->get_display_date_label( $document_display_date );
 	}
 
-	public function get_display_date_label( $date_string ) {
-
+	public function get_display_date_label( string $date_string ): string {
 		$date_labels = array(
-			'invoice_date'	=> __( 'Invoice Date' , 'woocommerce-pdf-invoices-packing-slips' ),
-			'order_date'	=> __( 'Order Date' , 'woocommerce-pdf-invoices-packing-slips' ),
+			'document_date' => __( 'Document Date', 'woocommerce-pdf-invoices-packing-slips' ),
```

Comment:
I think having "Invoice Date" is preferable, maybe using the document object title?